hutch_python/happi.py

```diff
@@ -62,7 +63,12 @@ def get_happi_objs(db, hutch):
             logger.warning("No items found in database for %s",
                            beamline.upper())
     # Instantiate the devices needed
-    dev_namespace = load_devices(*containers, pprint=False)
+    sig = inspect.signature(load_devices)
+    if "include_load_time" in sig.parameters:
+        kwargs = dict(include_load_time=True, load_time_threshold=0.5)
```

Comment:
0.5s is the default threshold right?  Though not strictly necessary, I think it's fine to include this here if only as a reminder of how to change the parameter.